index.js

```diff
@@ -0,0 +1,46 @@
+const later = require('later');
+const moment = require('moment-timezone');
+
+module.exports.timezone = (timezone) => {
```

Comment:
lets make it pass in later and timezone

`module.exports = (later, timezone) => {`

we should add some tests too